sdk/src/main/java/io/customer/sdk/util/DeepLinkUtil.kt

```diff
@@ -0,0 +1,61 @@
+package io.customer.sdk.util
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Build
+import android.util.Log
+import androidx.core.app.TaskStackBuilder
+
+interface DeepLinkUtil {
+    /**
+     * Creates list of intents for the provided link for the provided link.
+     *
+     * @param context reference to application context
+     * @param deepLink link to create intent for
+     * @return list of intents to add in [TaskStackBuilder] for the provided
+     * link, empty if no matching intent found
+     */
+    fun createIntentsForLink(context: Context, deepLink: String?): List<Intent>
+}
+
+class DeepLinkUtilImpl : DeepLinkUtil {
+    override fun createIntentsForLink(context: Context, deepLink: String?): List<Intent> {
+        val intent = deepLink?.let { link -> resolveDeepLinkIntent(context, link) }
+        return if (intent == null) {
+            Log.v(
+                TAG,
+                "No supporting application for this deepLink $deepLink," +
+                    " launching default activity"
+            )
```

Comment:
```suggestion
            logger.verbose(
                "No supporting application for this deepLink $deepLink," +
                    " launching default activity"
            )
```

The SDK has a `Logger` class that you inject into the constructor to do all logging with. We centralize it so we can log to files, log to logcat, etc.

Reply:
Thanks for sharing this, I'll update it to use `logger` instead